src/sentry/web/frontend/setup_wizard.py

```diff
@@ -143,6 +134,75 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
         context["organizations"] = list(org_mappings_map.values())
         return render_to_response("sentry/setup-wizard.html", context, request)
 
+    def post(self, request: HttpRequest, wizard_hash=None) -> HttpResponse:
+        """
+        This updates the cache content for a specific hash
+        """
+        organization_id = request.POST.get("organizationId", None)
+        project_id = request.POST.get("projectId", None)
+
+        if organization_id is None or project_id is None or wizard_hash is None:
+            return HttpResponseBadRequest()
+
+        mapping = get_object_or_404(OrganizationMapping, organization_id=organization_id)
```

Comment:
There is a missing validation if user can access this organization. It could be a security problem if we don't check for that.

You could get all the organizations of the user by running:
```
member_org_ids = OrganizationMemberMapping.objects.filter(
            user_id=request.user.id
        ).values_list("organization_id", flat=True)
```

And then this line would look like:
```
mapping = get_object_or_404(OrganizationMapping, organization_id=organization_id, organization_id__in=member_org_ids)
```
This will be `and` conditions checked if organization with requested id exists, and if it is in the organizations to which user has access to